backend/orchestrator/resources/dynamodb.ts

```diff
@@ -0,0 +1,23 @@
+export const PARTITION_KEY = 'pk';
+export const SORT_KEY = 'sk';
+
+export const OrchestratorTable = {
```

Comment:
We can leverage lift's [Database - DynamoDB Single Table construct](https://github.com/getlift/lift/blob/master/docs/database-dynamodb-single-table.md) to have less boilerplate and better DDB configuration out of the box

Reply:
I think we can use a more fine-grained permissions system for this table, by default Lift grants all permissions to all lambdas https://github.com/getlift/lift/blob/master/docs/database-dynamodb-single-table.md#permissions